### ISSUE TITLE
Can't unset auto_scaling

### ISSUE THREAD

**Issue by Ulrar** [mongodb/terraform-provider-mongodbatlas#1072]
### Terraform CLI and Terraform MongoDB Atlas Provider Version

```hcl
Terraform v1.3.8
on linux_amd64
+ provider registry.terraform.io/hashicorp/random v3.4.3
+ provider registry.terraform.io/hashicorp/vault v3.11.0
+ provider registry.terraform.io/mongodb/mongodbatlas v1.8.0
```

### Terraform Configuration File 

```hcl
resource "mongodbatlas_advanced_cluster" "cluster" {
...
}
```

### Steps to Reproduce
Create a cluster with auto scaling, then try to disable it

### Expected Behavior
Being able to

### Actual Behavior
- Removing the whole auto scaling bloc does nothing, no changes in the plan
- Trying to disable the fields manually like so : 
```hcl
      dynamic "auto_scaling" {
        for_each = local.is_tenant[each.value.name] ? [] : [{}]

        content {
          compute_enabled            = false
          compute_scale_down_enabled = false
          compute_min_instance_size  = ""
          compute_max_instance_size  = ""
        }
      }
```

Whatever I try, I get the following error on apply : 
`400 (request "COMPUTE_AUTO_SCALING_MIN_INSTANCE_SIZE_INVALID_FOR_DISABLED") Compute auto-scaling min instance size must be unset when scale down is disabled.`

The issue seems to be that these fields somehow default to their existing values, which means I can't unset them. Or at least I haven't figured out how, removing them doesn't do anything and setting them to empty values doesn't either.

### Additional Context

These clusters have been defined using the mongodb_cluster resource first, then imported later in advanced cluster, in case that makes a difference.

**Comment by Ulrar** [comment]
For anyone else who might have that issue, I just disabled auto scaling through the UI and re-ran an apply to update the state, worked fine.

**Comment by martinstibbe** [comment]
@Ulrar I was able to replicate using advanced cluster directly will track fix in engineering ticket [INTMDB-617](https://jira.mongodb.org/browse/INTMDB-617)

**Comment by Zuhairahmed** [comment]
just as an update @Ulrar this issue has been picked up for v1.8.2 release which should be published in the next month or two.

**Comment by Zuhairahmed** [comment]
hi @Ulrar v1.8.2 has been released which includes above fix. feel free to re-open if you need anything else. https://registry.terraform.io/providers/mongodb/mongodbatlas/latest

**Comment by Ulrar** [comment]
Thanks !
CC @pacoguzman

**Comment by pazsaragi** [comment]
For anyone else who reached this thread and is looking for the minimal version fix - I found that upgrading to version `1.11.0` did the trick 👍
